class RepositoryError(Exception):
    pass

class ValidateError(Exception):
    pass